CHANGELOG

```diff
@@ -1,3 +1,9 @@
+== 9.4.0 2017-02-03
+* Adds button to billing/shipping entry screens to fill address information from the other one.
+* Fixes and unifies view controller behavior around theming and nav bars.
+* Adds month validity check to `validationStateForExpirationYear`
+* Changes some Apple Pay images to better conform to official guidelines.
```

Comment:
yeah, it's definitely pretty minor, but probably worth calling out in case the UI change is surprising